lib/azure/armrest/model/base_model.rb

```diff
@@ -263,3 +263,4 @@ class Snapshot < BaseModel; end
 end
 
 require_relative 'storage_account'
+require_relative 'virtual_machine'
```

Comment:
@bzwei It's already defined in base_model.rb, but it's just a stub really. We already did something similar with StorageAccount.